webapp/src/containers/components/modals/deleteModal.tsx

```diff
@@ -0,0 +1,51 @@
+import React, {useEffect, useState} from 'react';
+
+import Button from 'react-bootstrap/Button';
+import Modal from 'react-bootstrap/Modal';
+
+import './styles.css';
+
+type Props = {
+    visible: boolean;
```

Comment:
```suggestion
    visibility: boolean;
```
Fix this in all the PRs. it should be visibility and setVisibility. Every var name should make proper sense. So that it clears the use of the code if any other dev is working on it. The names you have given also clear the meaning but it doesn't sounds good.

Reply:
Also, I think we removed this variable right?